CONTRIBUTING.md

```diff
@@ -46,6 +48,18 @@ This guide will help you get started and outline some things you should know whe
 * See [here](./documentation/development/testing.md) for how to run the tests
 
 
+## Developing
+
+* all dependencies are located in the [vendor](vendor) folder,
+  which is checked into Git
+* update dependencies: `glide up`
+* adding a new Go library:
+  * update dependencies in a separate PR
+  * `glide get <package name>`
+  * don't forget to add the updated glide files and vendor folder
+    in the pull request for the feature that requires the new library
```

Comment:
I doubt its really a matter of remembering. Aren't we are trying to let a user know that updating a dependency will result in changes to the vendor folder? Maybes something like
```
* your PR should contain updates to the glide files and vendor folder
``